src/Resources/Reporter.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace TestMonitor\Mantis\Resources;
+
+class Reporter extends Resource
+{
+    /**
+     * The id of the reporter.
+     *
+     * @var string
+     */
+    public $id;
+
+    /**
+     * The name of the reporter.
+     *
+     * @var string
+     */
+    public $name;
+
+    /**
+     * The real name of the reporter.
+     *
+     * @var string
+     */
+    public $real_name;
+
+    /**
+     * The reporter mail.
+     *
+     * @var string
+     */
+    public $email;
+
+    /**
+     * Create a new resource instance.
+     *
+     * @param $attributes
+     */
+    public function __construct(array $attributes)
```

Comment:
Now all the attributes are optional? At least what a reporter has is a id I asume? And very likely you get the name and mailladdress also?

https://documenter.getpostman.com/view/29959/mantis-bug-tracker-rest-api/7Lt6zkP#13f890da-50ff-0ab3-ff66-32b701d90c66